test/DynamoCoreWpfTests/CoreUITests.cs

```diff
@@ -797,6 +797,32 @@ public void WorkspaceContextMenu_IfSubmenuOpenOnMouseHover()
             Assert.IsTrue(currentWs.WorkspaceLacingMenu.IsSubmenuOpen);
         }
 
+        [Test]
+        [Category("UnitTests")]
+        public void TestDumpLibraryToXml()
+        {
+            ViewModel.DumpLibraryToXmlCommand.Execute(null);
+
```

Comment:
I think this test is not to test whether DumpLibrary.... method is callable. You must validate the output against existing data. For example, here you can verify if the XML file is created or not, and then how many elements are created is it same as the number of items in the library view? 

There should be a way to know the file name dumped, can we have a command completed event with output path as a parameter, so you don't need to iterate and guess the xml file name.